### PR TITLE
Added User and Article

### PR DESCRIPTION
**Pull request by IgorPrihodko**

### CODE REVIEW

File: src/main/java/controller/UserRegistrationServlet.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String email = req.getParameter("email");
+        String password = req.getParameter("password");
+        String repeatPassword = req.getParameter("repeatPassword");
+
+        if (!repeatPassword.equals(password)) {
+            resp.setStatus(HttpServletResponse.SC_CONFLICT);
+            System.out.println("Passwords are not equals");
```
Comment (Sofasmile) [inline src/main/java/controller/UserRegistrationServlet.java]: better write it in browser

File: src/main/java/controller/AddArticleServlet.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String title = req.getParameter("title");
+        String description = req.getParameter("description");
+        Double price = Double.parseDouble(req.getParameter("price"));
+
+        Article article = new Article(title, description, price);
+        DataBase.ARTICLES.put(article.getId(), article);
+        resp.setStatus(HttpServletResponse.SC_OK);
+        System.out.println(article);
```
Comment (Sofasmile) [inline src/main/java/controller/AddArticleServlet.java]: the same

File: src/main/java/controller/AddArticleServlet.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("addArticle.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String title = req.getParameter("title");
```
Comment (Sofasmile) [inline src/main/java/controller/AddArticleServlet.java]: better add checking if parameters is empty

Comment (IgorPrihodko) [reply src/main/java/controller/AddArticleServlet.java]: fixed

File: src/main/java/controller/UserRegistrationServlet.java (hunk 1 above)
Comment (IgorPrihodko) [reply src/main/java/controller/UserRegistrationServlet.java]: done

File: src/main/java/controller/AddArticleServlet.java (hunk 2 above)
Comment (IgorPrihodko) [reply src/main/java/controller/AddArticleServlet.java]: done

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**